plugin/src/main/java/org/xtclang/plugin/tasks/XtcCompileTask.java

```diff
@@ -219,12 +213,18 @@ public void executeTask() {
             args.add("-r", processedResourcesDir.getAbsolutePath());
         }
 
+        final var shouldRebuild = getRebuild().get();
+        if (!shouldRebuild && overrideVerboseLogging) {
```

Comment:
Fixing. The outermost resolution is enough. I put this in because you can actually run with defaults and disable rebuild at more fine grained levels, and may not intend to, which actually was the case for the manualTests compile tasks for a bit.